templates/santa/finish.html.twig

```diff
@@ -3,41 +3,24 @@
 {% block content %}
     <div class="content result-block">
         {% if secretSanta.isDone %}
-            <h2>Well done! All messages were sent</h2>
+            <h2>Well done! All messages were sent, happy gifting!</h2>
 
             <p class="summary">
-                You have received a private message explaining how to retrieve the santa repartition. You should probably don't look at it, to avoid breaking the Secret Santa spirit.
+                You have received a private message explaining how to retrieve the santa repartition. 
+                You should probably don't look at it, to avoid breaking the Secret Santa magic.
             </p>
         {% else %}
-            <h2>A technical error occurred when sending messages to users</h2>
+            <h2>😱 It takes too much time to send all the messages!</h2>
```

Comment:
I do not really agree with this phrase as main title because errors can also be due to lot of different problems

Reply:
How can we rephrase then?

> Wait! All the messages are not sent yet!

> Oh! Oh! Oh! Santa did not send all the messages yet!

> All the messages are not sent yet, please read carefully

Reply:
ping @pyrech :+1:

Reply:
I like the first message :+1: